specification/network/resource-manager/Microsoft.Network/stable/2021-01-01/loadBalancer.json

```diff
@@ -1635,6 +1697,13 @@
           "$ref": "./network.json#/definitions/SubResource",
           "description": "A reference to a pool of DIPs. Inbound traffic is randomly load balanced across IPs in the backend IPs."
         },
+        "backendAddressPools": {
+          "type": "array",
+          "items": {
+            "$ref": "./network.json#/definitions/SubResource"
+          },
+          "description": "An array of references to pool of DIPs."
+        },
```

Comment:
Can `backendAddressPools` property replace `backendAddressPool`?

Reply:
It depends on the situation. BackendPools here will only be used for special scenario for our features. BackendAddressPool will be empty

Reply:
> It depends on the situation. BackendPools here will only be used for special scenario for our features. BackendAddressPool will be empty

Can you specify when we need to use Pools and when we need to use Pool? Because it's important for CLI and powershell developers

Reply:
@msyyc FYI